src/backend/utils/authUtils.js

```diff
@@ -0,0 +1,22 @@
+import { Response } from "miragejs";
+import dayjs from "dayjs";
+import jwt_decode from "jwt-decode";
+
+export const requiresAuth = function (request) {
+  const encodedToken = request.requestHeaders.authorization;
+  const decodedToken = jwt_decode(
+    encodedToken,
+    process.env.REACT_APP_JWT_SECRET
```

Comment:
why you decode token here

Reply:
This is a backend code, it came with the mockbee setup. It verifies whether the token sent by the app is valid or not

Reply:
okay